mqtt/mqtt-bridge/src/controller/mod.rs

```diff
@@ -82,9 +82,13 @@ impl Sidecar for BridgeController {
         }
 
         loop {
-            let wait_bridge_or_pending = if bridges.is_terminated() {
-                // if no active bridges available, wait only for a new messages arrival
-                Either::Left(future::pending())
+            let wait_bridge_or_pending = if (bridges.is_terminated() || bridges.is_empty()) {
```

Comment:
I guess there should warnings for the redundant parenthesizes for if-expresssions.